realm/realm-library/src/androidTest/java/io/realm/DynamicRealmTests.java

```diff
@@ -664,4 +668,18 @@ public void run() throws Exception {
             }
         });
     }
+
+    @Test
+    public void equalTo_noFieldObjectShouldThrow() {
```

Comment:
No, the issue was about a non-existing field.
For a non-existing table, our exception message is correct, i did check it.